src/core/bitop.d

```diff
@@ -706,3 +706,55 @@ version (D_InlineAsm_X86_64)
         }
     }
 }
+
+/**
+ *  Bitwise rotates unsigned integer types
```

Comment:
It's better to use `rotate` here, b/c the function doesn't do anything on its own account.